spec/features/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Users' do
+  let(:returning_user) { FactoryBot.create(:user) }
+  let(:new_event) { FactoryBot.create(:event) }
+
+  def new_user
+    { name: Faker::Name.name, email: Faker::Internet.email }
+  end
+
+  def event
+    {
+      name: Faker::Address.building_number,
+      date: Faker::Date.between(from: 1.year.ago, to: 1.year.from_now),
+      description: Faker::Lorem.sentence
+    }
+  end
+
+  def sign_up(new_user)
+    within '.column' do
+      fill_in 'user[name]', with: new_user[name]
+      fill_in 'user[email]', with: new_user[email]
+
+      click_button 'Sign up'
+    end
```

Comment:
What next? You must expect the user to be created into the database and after clicking on the sign up button, a new page appears, routing has been done. you should expect all those things.